guides/documentations/configuration.md

```diff
@@ -0,0 +1,91 @@
+# How to set your plugin's configuration
+
+The DefaultConfiguration class is used to set your plugin's configuration.
```

Comment:
Reput the short explanation of DefaultPlugin with example of code that set configuration.
before this line.